package-lock.json

```diff
@@ -0,0 +1,3017 @@
+{
```

Comment:
We are using `pnpm` here, please remove this.

Reply:
fixed